app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Users index are coming soon!</h1>
+<h1>Users index</h1>
+<div class="users-container">
+  <% @users.each do |user| %>
+    <%= link_to user_path(user), :class => 'user-button' do %>
+      <div class="user-card">
+         <%= image_tag("user.webp", :class => "user-image", :alt => "user-image") %>
+        <div class="user-data">
+          <h1><%= user.name %></h1>
+          <span>Number of posts: <%= user.posts_counter %></span>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
```

Comment:
- [ ] Kindly follow the given wireframe, so as to position your content as in the given wireframe

Your design
![image](https://user-images.githubusercontent.com/81932478/216192178-c43dcc84-e428-44b2-90ac-172a2b59d464.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/216192234-3c3ff58b-ba42-4a65-a923-81491e908541.png)